pytest.ini

```diff
@@ -12,4 +12,4 @@ log_file_level = INFO
 markers =
     unit: quick tests that do not require a solver, must run in <2s
     component: quick tests that may require a solver
-    integration: long duration tests
+    integration: long duration tests
```

Comment:
add new line, currently git is saying you are changing the file because there is no newline at the end of the file